Problem: autocli use wrong clientCtx

yihuang opened the pull request:
Solution:
- update dependency to include the fix, see https://github.com/cosmos/cosmos-sdk/pull/19060

<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
v                               ✰  Thanks for creating a PR! ✰    
v    Before smashing the submit button please review the checkboxes.
v    If a checkbox is n/a - please still include it but + a little note why
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

Closes: #XXX

## Description

<!-- Add a description of the changes that this PR introduces and the files that
are the most critical to review.
-->

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

mmsqe commented on tests/integration_tests/utils.py:
```diff
@@ -351,4 +351,4 @@ def cb(attrs):
     wait_for_block_time(cli, isoparse(proposal["voting_end_time"]))
     proposal = cli.query_proposal(proposal_id)
-    assert proposal["status"] == status, proposal
+    assert proposal["status"] == "PROPOSAL_STATUS_PASSED", proposal
```
we need keep both  number and string enum check right?

yihuang replied:
why?

mmsqe commented on tests/integration_tests/utils.py:
```diff
@@ -351,4 +351,4 @@ def cb(attrs):
     wait_for_block_time(cli, isoparse(proposal["voting_end_time"]))
     proposal = cli.query_proposal(proposal_id)
-    assert proposal["status"] == status, proposal
+    assert proposal["status"] == "PROPOSAL_STATUS_PASSED", proposal
```
maybe changed, I rmb newer version use number

Commit:
Problem: autocli use wrong clientCtx

Solution:
- update dependency to include the fix, see https://github.com/cosmos/cosmos-sdk/pull/19060

changelog

fix test
